src/react-diagrams/state/DefaultState.ts

```diff
@@ -27,7 +27,7 @@ export class DefaultState extends State<DiagramEngine> {
     this.dragCanvas = new DragCanvasState();
     this.dragItems = new DragDiagramItemsState();
     this.createLink = customCreateLink ?? new CreateLinkState();
```

Comment:
I think if we do not provide the customCreateLink we should use the CreateLinkState we have in MetaDiagrams as the default one

Reply:
Nice. I didn't know that syntax :+1: